Makefile.in

```diff
@@ -1222,6 +1227,8 @@ stamps/build-llvm-newlib: $(LLVM_SRCDIR) $(LLVM_SRC_GIT) $(BINUTILS_SRCDIR) $(BI
 	cp $(notdir $@)/lib/LLVMgold.$(SHARED_LIB_SUFFIX) $(INSTALL_DIR)/lib
 	cd $(INSTALL_DIR)/bin && ln -s -f clang $(NEWLIB_TUPLE)-clang && \
 	    ln -s -f clang++ $(NEWLIB_TUPLE)-clang++
+	cd $(INSTALL_DIR)/bin && echo "-march=${MARCH} -mabi=${MABI} -mtune=${MTUNE}" >> $(LINUX_TUPLE)-clang.cfg \
```

Comment:
`$(NEWLIB_TUPLE)` instead of `$(LINUX_TUPLE)`.
Same for the line below.

Reply:
Why do you have a trailing `\`? Please remove it.

Reply:
Why do you append with `>>` and not `>` overwrite?
Appending could result in multiple entries (in case a user builds twice without manually deleting the installation directory). So, I prefer overwrite.

Reply:
> `$(NEWLIB_TUPLE)` instead of `$(LINUX_TUPLE)`. Same for the line below.

Sorry for that, it's a copy-paste error. :-)